wpilibc/athena/include/DriverStation.h

```diff
@@ -68,6 +68,7 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   Alliance GetAlliance() const;
   int GetLocation() const;
   void WaitForData();
+  bool WaitForData(long long timeout);
```

Comment:
To answer a question like this, one could pull all of the java code listed at https://firstwiki.github.io/wiki/robot-code-directory and grep for it... that's probably 100 teams or so.